lib/default.nix

```diff
@@ -79,7 +79,7 @@ let
       recursiveUpdate matchAttrs overrideExisting getOutput getBin
       getLib getDev getMan chooseDevOutputs zipWithNames zip
       recurseIntoAttrs dontRecurseIntoAttrs;
-    inherit (self.lists) singleton forEach foldr fold foldl foldl' imap0 imap1
+    inherit (self.lists) singleton forEach fold foldr foldl foldl' imap0 imap1
```

Comment:
Is this supposed to just change the position of foldr?

Reply:
Ah, good catch, I'll revert this.